packages/@interaxyz/mobile/src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -19,11 +20,24 @@ function OnboardingSuccessScreen() {
 
   const { t } = useTranslation()
 
+  const assetsConfig = getAppConfig().themes?.default?.assets
+
+  const image = assetsConfig?.onboardingSuccessImage
+
   return (
     <View style={styles.container}>
-      <Image source={background} style={styles.backgroundImage} />
-      <Logo color={colors.contentOnboardingComplete} size={70} />
-      <Text style={styles.text}>{t('success.message')}</Text>
+      {image ? (
+        <>
+          <Image source={image} />
+          <Text style={styles.textWithImage}>{t('success.message')}</Text>
+        </>
+      ) : (
+        <>
+          <Image source={background} style={styles.backgroundImage} />
+          <Logo color={colors.contentOnboardingComplete} size={70} />
+          <Text style={styles.textWithBackground}>{t('success.message')}</Text>
+        </>
+      )}
```

Comment:
this works, but i wonder if we want to consider customising the background and image separately, to be consistent with the welcome screen? we could use the Logo as the image if it is undefined as well?